lib/workers/global/config/parse/cli.ts

```diff
@@ -114,6 +115,22 @@ export function getConfig(input: string[]): AllConfig {
         if (option.cli !== false) {
           if (opts[option.name] !== undefined) {
             config[option.name] = opts[option.name];
+            if (option.name === 'dryRun') {
+              if (
+                config[option.name] === 'false' ||
+                config[option.name] === 'null'
```

Comment:
i think we should not warn for null or we can't disable via cli when enabled via file or env.

Reply:
should i delete the log msg ?

Reply:
we should warn for false and true.